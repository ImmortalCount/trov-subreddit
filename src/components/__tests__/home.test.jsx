import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../home'

describe('Home Component', () => {
    it('Home matches snapshot', () => {
        const modalRender = renderer.create(
            <Home />
        ).toJSON()
        expect(modalRender).toMatchSnapshot()
    })
})
