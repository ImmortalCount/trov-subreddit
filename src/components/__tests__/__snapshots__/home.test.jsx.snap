// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Home matches snapshot 1`] = `<div />`;
